src/typechecker/FStar.TypeChecker.Rel.fs

```diff
@@ -1476,7 +1504,7 @@ let rec really_solve_universe_eq pid_orig wl u1 u2 =
                                         (Print.univ_to_string u2))
 
         | U_name x, U_name y ->
-          if (string_of_id x) = (string_of_id y)
+          if (string_of_id x) = (string_of_id y) || Options.ml_ish ()
```

Comment:
this is slightly surprising? Why the ml_ish here?

Reply:
Thanks for catching it. At some point bootstrapping was failing so I added it, removed now.